### ISSUE TITLE
Add support for `cy.callFunction()`

### ISSUE THREAD

**Issue by gregfenton** [prescottprue/cypress-firebase#141]
The ability to call a Cloud Function would be powerful.

A key benefit would be the ability to do "administrative functions" via those cloud functions.  Say, for example, deleting all existing users in a FB project.

https://firebase.google.com/docs/functions/callable#call_the_function

**Comment by prescottprue** [comment]
I could see this being a feature for sure, but something to keep in mind is that if this is planned to be used for cleanup, like you were mentioning, that may not always get run depending on where the logic is placed. Also, Cypress advocates for only cleaning up when absolutely necessary as mentioned in [their Best Practices docs](https://docs.cypress.io/guides/references/best-practices.html#Is-resetting-the-state-necessary)

**Comment by gregfenton** [comment]
I agree with 98% of the Cypress best practices.  Some of their best practices assume a db that can be simply cleared and restored.  There is friction when it comes to working with a more rigid environment like Firebase.  Firebase Auth, in particular, don't work that way 😄 

I'm not looking to tear down & build up the environment with every single test run.  But even the one-time setup needs to be automated for consistency.  Adding dozens of users, their roles, their groups....don't want to do that manually.  And I don't want to share my dev & test environments with others.

**Comment by prescottprue** [comment]
That is fair, I definitely feel you there! Something to note though is that firebase-tools does have import capability for the Firestore emulator (through the --import flag when booting up emulator with `emulators:exec` or `emulators:start` as well as for auth ([auth import/export](https://firebase.google.com/docs/cli/auth))

**Comment by prescottprue** [comment]
Here is an example of what I was mentioning with firebase-tools:

```bash
#!/bin/sh
# NOTE: firebase-extra is used since firebase-tools doesn't currently support emulators (see https://github.com/firebase/firebase-tools/issues/1957)
# NOTE: Make sure to set FIRESTORE_EMULATOR_HOST and FIREBASE_DATABASE_EMULATOR_HOST to environment to use emulators

echo "Seeding DB..."

# Update data in firestore with inline data
firebase-extra firestore:set /users/ABC123 -d '{"some": "data"}' && echo "Updated user in firestore"

# Seed data to RTDB from a file
firebase-extra database:update /roles ./data/roles.json && echo "Set roles from file"

echo "Successfully seeded databases"
```

**Comment by prescottprue** [comment]
Of note: as of [firebase-tools v8.5.0](https://github.com/firebase/firebase-tools/releases/tag/v8.5.0) there is now support for using `--import` with RTDB

**Comment by gregfenton** [comment]
And [v8.6.0](https://github.com/firebase/firebase-tools/releases/tag/v8.6.0) adds `--export-on-exit`.  Things in FB Emulator space are maturing!
